packages/zero-client/src/client/context.ts

```diff
@@ -34,16 +37,29 @@ export class ZeroContext implements QueryDelegate {
   readonly #batchViewUpdates: (applyViewUpdates: () => void) => void;
   readonly #commitListeners: Set<CommitListener> = new Set();
 
+  readonly slowMaterializationTheshold: number | undefined;
```

Comment:
typo

Reply:
? I don't see the typo. Changed name anyway.